roles/openshift_etcd_facts/vars/main.yml

```diff
@@ -5,6 +5,7 @@ etcd_hostname: "{{ openshift.common.hostname }}"
 etcd_ip: "{{ openshift.common.ip }}"
 etcd_cert_subdir: "etcd-{{ openshift.common.hostname }}"
 etcd_cert_prefix:
-etcd_cert_config_dir: "{{ '/etc/etcd' if not openshift.common.is_etcd_system_container | bool else '/var/lib/etcd/etcd.etcd/etc' }}"
+etcd_cert_config_dir: "/etc/etcd"
+etcd_system_container_cert_config_dir: "{{ r_etcd_common_system_container_host_dir }}/etc"
```

Comment:
That will not work unless etcd_common is a dependency of openshift_etcd_facts. ``etcd_system_container_cert_config_dir: /var/lib/etcd/etcd.etcd/etc`` will do.